immscore/imms.cc

```diff
@@ -172,7 +172,8 @@ void Imms::start_song(int position, string path)
     if (!current.isanalyzed())
     {
         string epath = rex.replace(path, "'", "'\"'\"'", Regexx::global);
-        system(string("analyzer '" + epath + "' &").c_str());
+        int result = system(string("analyzer '" + epath + "' &").c_str());
+        (void)(result);
```

Comment:
This is so the compiler doesn't complain (and fail because -Werror is on) about unused return values.

Reply:
I know what it's for; however, ignoring the error is not good (which is why we get a warning). It would be better to log it and/or print it, with perror().